src/Contracts/ResponseInterface.php

```diff
@@ -5,44 +5,47 @@
 interface ResponseInterface
 {
     /**
-     * Get all messages
-     *
-     * messages can be
+     * Get all associated fraud messages
      *
      * @return \Omnifraud\Contracts\MessageInterface[]
      */
     public function getMessages(): array;
 
     /**
-     * How trustworthy is that transaction, 100 is best, 0 is worst
+     * The score assigned by the service, where 100 is the best and 0 is the
+     * worst. In the event that the response is pending, this is nullable.
      *
      * @return float|null
      */
-    public function getPercentScore(): ?float;
+    public function getScore(): ?float;
```

Comment:
I removed the `percent` from the name because I don't think it adds any value. Score is quite self explanatory and not necessarily a percentage just because of the 0-100 range.